sigma-schema.json

```
@@ -0,0 +1,246 @@
+      "type": "string",
+      "description": "Last modification date of the rule. Use the format YYYY/MM/DD",
+      "pattern": "^\\d{4}[/]\\d{2}[/]\\d{2}$"
+    },
+    "logsource": {
+      "type": "object",
+      "description": "The log source that the rule is supposed to detect malicious activity in.",
+      "properties": {
+        "category": {
+          "type": "string",
+          "description": "Select all log files written by a certain group of products, like firewalls or web server logs"
+        },
+        "product": {
+          "type": "string",
+          "description": "Select all log outputs of a certain product, e.g. all Windows Eventlog types including \"Security\", \"System\", \"Application\" and the new log types like \"AppLocker\" and \"Windows Defender\""
+        },
+        "service": {
+          "type": "string",
+          "description": "Select only a subset of a product's logs, like the \"sshd\" on Linux or the \"Security\" Eventlog on Windows systems"
+        },
+        "definition": {
+          "type": "string",
+          "description": "Describe the log source, including some information on the log verbosity level or configurations that have to be applied"
+        }
+      }
+    },
+    "detection": {
+      "type": "object",
+      "required": ["condition"],
+      "description": "A set of search-identifiers that represent properties of searches on log data",
+      "additionalProperties": {
+        "anyOf": [
+          {
+            "type": "array",
+            "items": {
+              "anyOf": [
+                {
+                  "type": "string"
+                },
+                {
+                  "type": "integer"
+                },
+                {
+                  "type": "object",
+                  "items": {
+                    "type": "string"
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "type": "object",
+            "items": {
+              "type": "string"
+            }
+          }
+        ]
+      },
+      "properties": {
+        "condition": {
+          "anyOf": [
+            {
+              "type": "string",
+              "description": "A search condition that is applied to the log data. The following format must be used: fieldname : value"
+            },
+            {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "minLength": 2
+              }
+            }
+          ],
+          "description": "A set of search-identifiers that represent properties of searches on log data"
+        },
+        "timeframe": {
+          "type": "string",
+          "description": "Is a special Search-Identifier used only with Aggregation conditions Defines a time period in which the aggregation should be applied. The following format must be used: number + letter (in lowercase) - Xs seconds - Xm minutes - Xh hours - Xd days"
+        }
```

Since this field was never really fully implemented. Its best not to mention it at all. Especially since in V2 its was removed.
```suggestion
        }
```